app/forms/support_interface/conditions_form.rb

```diff
@@ -73,8 +89,84 @@ def save
       ).save!
     end
 
+    def subject_name
+      subject&.name
+    end
+
+    def subject
+      application_choice.current_course&.subjects&.first
+    end
+
+    def cutoff_date
+      application_choice.current_course.start_date - 5.years
```

Comment:
what knowledge does this represent?

Reply:
Good question. This duplicates similarly mysterious code in `OfferWizard`. I've extracted it to `Course#ske_graduation_cutoff_date` which hopefully makes the intention a bit more explicit.